xbmc/interfaces/json-rpc/InputOperations.cpp

```diff
@@ -84,6 +86,32 @@ JSONRPC_STATUS CInputOperations::ExecuteAction(const std::string &method, ITrans
   return SendAction(action);
 }
 
+JSONRPC_STATUS CInputOperations::ButtonEvent(const std::string& method,
+                                             ITransportLayer* transport,
+                                             IClient* client,
+                                             const CVariant& parameterObject,
+                                             CVariant& result)
+{
+  std::string button = parameterObject["button"].asString();
+  std::string keymap = parameterObject["keymap"].asString();
+  int holdtime = (int)parameterObject["holdtime"].asInteger();
```

Comment:
Sorry for another style request - please use the c++ version, `static_cast<int>()`

Reply:
In fact this cast wasn't needed at all.